inst/examples/keras/train.R

```diff
@@ -20,6 +20,8 @@ x_test <- x_test / 255
 y_train <- to_categorical(y_train, 10)
 y_test <- to_categorical(y_test, 10)
 
+backend()$set_learning_phase(TRUE)
```

Comment:
It seems like we shouldn't need this? (nowhere else do we document that export_savedmodel requires a call to set_learning_phase(TRUE). It seems like if this is the case then this needs to be prominently stated in multiple places

Reply:
Yes, this is needed since it's a placeholder `keras` seems to use as a flag, not specifying this causes the model to require this placeholder as a parameter over each prediction. There is a bit more info in https://keras.io/backend/#learning_phase.

Reply:
Okay, you can actually use the `k_set_learning_phase()` function here rather than accessing the `backend()`

Reply:
Great, changed.

Reply:
One other thing to consider: I have a feeling that it's important to manually set the learning phase to 0 if the script is going to do any evaluation (otherwise when evaluation data passes through the graph it will be used to update weights, etc.). Not 100% sure about this though, let's do a bit of research on this and then update the `export_savedmodel()` docs for Keras and the inline comments to reflect what we find out.